Dangerfile

```diff
@@ -52,6 +52,8 @@ end
 podspec_updated = !git.modified_files.grep(/Moya.podspec/).empty?
 cartfile_updated = !git.modified_files.grep(/Cartfile/).empty?
 cartfile_resolved_updated = !git.modified_files.grep(/Cartfile.resolved/).empty?
+package_updated = !git.modified_files.grep(/Package.swift/).empty?
+package_resolved_updated = !git.modified_files.grep(/Package.resolved/).empty?
 
 if podspec_updated && (!cartfile_updated || !cartfile_resolved_updated)
```

Comment:
Would you be up for adding SPM to the checks here as well? Seems like we check if Carthage was updated and CP not or CP was updated and Carthage not.

Reply:
Yeah, I think this could be a beneficial addition. If we don't add it here, let's open an issue for it